arcade-services.slnx

```diff
@@ -49,6 +49,7 @@
     <Project Path="test/Microsoft.DotNet.Maestro.Tasks.Tests/Microsoft.DotNet.Maestro.Tasks.Tests.csproj" />
     <Project Path="test/ProductConstructionService.Api.Tests/ProductConstructionService.Api.Tests.csproj" />
     <Project Path="test/ProductConstructionService.BarViz.Tests/ProductConstructionService.BarViz.Tests.csproj" />
+    <Project Path="test/ProductConstructionService.Cli.Tests/ProductConstructionService.Cli.Tests.csproj" Id="d58d01e5-45e5-4969-827c-6893f2fba261" />
```

Comment:
What's up with the Id?

Reply:
got no idea, it was added there automatically, will remove it